lib/tasks/next_available_error.rake

```diff
@@ -0,0 +1,8 @@
+require 'util/error_code'
+
+namespace :error_code do
+  task :next do
+    error_code = Util::ErrorCode.new('./app/domain/errors.rb')
```

Comment:
Something I missed: we should avoid `Util`.  Do we already have an `Error` module or `Rake` module?  I'd put it in there instead.